README.md

```diff
@@ -44,9 +44,11 @@ This file contains a number of front-end interview questions that can be used wh
 * If you could master one technology this year, what would it be?
 * What are the differences between Long-Polling, Websockets and SSE?
 * Explain the importance of standards and standards bodies.
-* What is FOUC? How do you avoid FOUC?
+* What is Flash of Unstyled Content (FOUC)? How do you avoid FOUC?
 * Do your best to describe the process from the time you type in a website's URL to it finishing loading on your screen.
-* Explain what ARIA and screenreaders are, and how to make a website accessible.
+* What is your approach to accessiblity and developing in keyboard accessibility and visual focus, such as using foundational HTML and Javascript techniques? 
+* Explain your experience with Web Accessibility Initiative's Accessible Rich Internet Applications (WAI-ARIA): Do you know the difference between developing an aceesible website following Web Content Accessibility Guidelines (WCAG) web standards and custom developing for assistive technologies such as screenreaders with WAI-ARIA?
```

Comment:
What I always try to determine from any new contribution is what kind of skills or knowledge you're trying to vet from a candidate through your question(s). As you note, you want to see if someone has knowledge of accessibility development techniques and standards. That said, the first two questions could, indeed, be simplified to something along the lines of `Do you know how to develop with WAI-ARIA or WCAG standards in mind?` or `What is your experience developing with WAI-ARIA or WCAG standards in mind?`. Acronyms or no acronyms, the rest of the wording is a bit too verbose.